Cargo.toml

```diff
@@ -57,7 +57,7 @@ clickhouse-rs = { git = "https://github.com/spiceai/clickhouse-rs.git", tag = "0
 datafusion = "41.0.0"
 datafusion-federation = "0.1"
 datafusion-federation-sql = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "ae9da45f8b90451119f3ca032b75ad82a28d7547" }
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "2c7522fa2f37ac5e07691ce18d0b13c5b1bd1a7d" }
```

Comment:
What's this for?

Reply:
This brings in the latest changes for DuckDB `ATTACH` support, for federation